src/LostAndFound/GameManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace LostAndFound
+{
+    public static class GameManager
+    {
+        public static void StartGame()
+        {
+            PrintIntro();
+        }
+
+        private static void PrintIntro()
+        {
+            TimedEvents.TimedWrite(
+                "\n\n\t\tLost and Found..." +
+                "\n\t\t\tThe Ultimate Survival Game..." +
+                "\n\n\t\t\tWritten and Developed by: Zach Hollis\n\n",
+                TimedEvents.WriteSpeed.Slow);
```

Comment:
Personally, I find it hard to tell the difference where we are passing separate args and where we are doing string concatenation. So I would recommend building the string separately from the function call.

```suggestion
            var introText = 
                "\n\n\t\tLost and Found..." +
                "\n\t\t\tThe Ultimate Survival Game..." +
                "\n\n\t\t\tWritten and Developed by: Zach Hollis\n\n";
                
            TimedEvents.TimedWrite(introText, TimedEvents.WriteSpeed.Slow);
```

Reply:
Also we need to add your name to the intro credits. We should talk about what we want it to say.